src/web/mjs/connectors/MangaDex.mjs

```diff
@@ -231,4 +224,26 @@ export default class MangaDex extends Connector {
         });
         return range.join('-');
     }
+
+    fetchJSON( request, retries ) {
```

Comment:
Maybe use `fetchJSON(uri, retries)` instead, and always composing the `Request` object from the given `uri` parameter within the function itself, this saves many `new Request()` instantiations before calling `fetchJSON()`